src/nica/b64enc.c

```diff
@@ -0,0 +1,153 @@
+/*
+ * This file is part of libnica.
+ *
+ * Copyright © 2016-2017 Intel Corporation
+ *
+ * libnica is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1
+ * of the License, or (at your option) any later version.
+ */
+
+#define _GNU_SOURCE
+#define B64_LINE_LEN 76
```

Comment:
Can you move this define after the header includes? It would look better to be placed next to the arrays you've initialized below.